app/models/conference.rb

```diff
@@ -132,6 +132,73 @@ def date_range_string
   end
 
   ##
+  # Builds out the model for returning a json response
+  #
+  #
+  def as_json(_options={})
```

Comment:
You do know about [serialization](http://api.rubyonrails.org/classes/ActiveModel/Serialization.html) right?